src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableTestWarningSeverity.java

```diff
@@ -0,0 +1,21 @@
+/*
+UnusedLocalVariable
+severity = warning
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unusedlocalvariable;
+
+public class InputUnusedLocalVariableTestWarningSeverity {
+
+    void m() {
+     @Test.A Outer p1 = new @Test.A Outer(); // ok
+     @Test.A Outer.@Test.B Inner p2 = p1.new @Test.B Inner();
+     // violation above but severity is set to warning
```

Comment:
Remove `but severity is set to warning`

Violation is pointer to what user asker to point, severity of it is different issue.

Reply:
@romani DONE

Note that due to ``severity = warning`` there are no violations in this file. Hence, I just fixed that.